helm/harvester/charts/harvester/templates/configmap.yaml

```diff
@@ -57,3 +74,16 @@ binaryData:
     {{ $.Files.Get $path | b64enc }}
   {{- end }}
 
+{{- if .Values.pilotcerts.enabled }}
```

Comment:
Do you really need a separate configmap just for certs? Isn't it enough to put them also in sandbox? They are copied to /data/harvester by default.